Codigo/ReferenceUtils.bas

```diff
@@ -144,3 +144,19 @@ Public Function GetName(ByRef SourceRef As t_AnyReference) As String
         GetName = NpcList(SourceRef.ArrayIndex).name
     End If
 End Function
+
+Public Function SetStatusMask(ByRef TargetRef As t_AnyReference, ByVal Mask As Long)
```

Comment:
It should be a `sub`, the function does not currently return any values and has no return type